### ISSUE TITLE
Force tokens to be available to all slugcats

### ISSUE THREAD

**Issue by SaltiestSyrup** [SaltiestSyrup/RWRandomizer#8]
Arena tokens and spearmaster broadcasts should have an option to become available to everyone regardless of normal accessibility rules.
This should be done by hooking `LoadPlacedObjects` in `RoomSettings` and forcing it to ignore slugcat requirements for tokens

**Comment by alphappy** [comment]
APWorld support contingent on alphappy/ArchipelagoRW#44 - not all tokens can be forced to appear trivially.

**Comment by SaltiestSyrup** [comment]
Is this still desired?

If this is implemented, there needs to be some new way for the player to determine what checks are accessible. External maps would no longer be enough if every token can be collected by everyone.
